libs/expo/shared/ui-components/src/lib/LibraryModal/Camera.tsx

```diff
@@ -0,0 +1,187 @@
+import { ReactNativeFile } from '@monorepo/expo/shared/apollo';
```

Comment:
why do we need this and `libs/expo/shared/ui-components/src/lib/CameraPicker/index.tsx`?

Reply:
updated camera for interaction and doc uploads. 
Now we have Camera component. but as each has different modals the modals will be separate. 
please take a look